### ISSUE TITLE
compile with `-pthread` succeeded but the htmlpage no longer runs my code unless i remove `-pthread`

### ISSUE THREAD

**Issue by Ma-Y** [emscripten-core/emscripten#17375]
Please include the following in your bug report:

**Version of emscripten/emsdk:**
Please include the output `emcc -v` here
emcc 3.1.15

**Failing command line in full:**
If this is compile or link-time failure please include the full failing command
along with its entire output.

`
emcc main.c -std=c99  -o index.html -pthread  -sPTHREAD_POOL_SIZE=navigator.hardwareConcurrency -lwebsocket.js -sASYNCIFY -sSINGLE_FILE -sFULL_ES3 -s USE_GLFW=3 
`

if i remove 
`-pthread  -sPTHREAD_POOL_SIZE=navigator.hardwareConcurrency`

everything works as normal.

platform Windows

**Comment by sbc100** [comment]
Yes this looks like https://emscripten.org/docs/porting/pthreads.html?highlight=thread#blocking-on-the-main-browser-thread.    The issue is that new threads don't get created while the main thread is blocked.  Two possible workarounds are either to use `PROXY_TO_PTHREAD` or `PTHREAD_POOL_SIZE`.
